src/amr/data/field/initializers/field_user_initializer.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef _PHARE_AMR_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_
+#define _PHARE_AMR_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_
```

Comment:
one more comment, I don't feel like this should be in amr, but more in `src/core/data/field` , no ?

Reply:
the initializers in my PR were in AMR as they were using samrai classes, but this could be in core alright